static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -74,63 +85,39 @@ const LLMMonitoringSection = lazy(
   () => import('sentry/components/events/interfaces/llm-monitoring/llmMonitoringSection')
 );
 
-export type GroupEventDetailsContentProps = {
+export interface EventDetailsContentProps {
   group: Group;
   project: Project;
   event?: Event;
-};
-
-type GroupEventEntryProps = {
-  entryType: EntryType;
-  event: Event;
-  group: Group;
-  project: Project;
-  sectionKey: FoldSectionKey;
-};
-
-function GroupEventEntry({
-  event,
-  entryType,
-  group,
-  project,
-  ...props
-}: GroupEventEntryProps) {
-  const organization = useOrganization();
-  const matchingEntry = event.entries.find(entry => entry.type === entryType);
-
-  if (!matchingEntry) {
-    return null;
-  }
-
-  return (
-    <EventEntry
-      projectSlug={project.slug}
-      group={group}
-      entry={matchingEntry}
-      {...{organization, event}}
-      {...props}
-    />
-  );
 }
 
-export function DefaultGroupEventDetailsContent({
+export function EventDetailsContent({
   group,
   event,
   project,
-}: Required<GroupEventDetailsContentProps>) {
+}: Required<EventDetailsContentProps>) {
   const organization = useOrganization();
   const location = useLocation();
   const hasNewTimelineUI = useHasNewTimelineUI();
   const hasStreamlinedUI = useHasStreamlinedUI();
   const tagsRef = useRef<HTMLDivElement>(null);
+  const eventEntries: {[key in EntryType]?: Entry} = useMemo(() => {
+    const {entries = []} = event;
+    return entries.reduce((entryMap, entry) => {
+      entryMap[entry.type] = entry;
+      return entryMap;
+    }, {});
+  }, [event]);
```

Comment:
move the type inside the reduce otherwise the reduce is adding items to `{}` and its being cast after
```suggestion
  const eventEntries = useMemo(() => {
    const {entries = []} = event;
    return entries.reduce<{[key in EntryType]?: Entry}>((entryMap, entry) => {
      entryMap[entry.type] = entry;
      return entryMap;
    }, {});
  }, [event]);
```